qa-ibd.planx-pla.net/manifests/fence/fence-public-config.yaml

```
@@ -0,0 +1,218 @@
+APP_NAME: 'Gen3 Data Commons'
+# Where fence microservice is deployed
+BASE_URL: 'https://qa-ibd.planx-pla.net/user'
+
+# flask's debug setting
+# WARNING: DO NOT ENABLE IN PRODUCTION (for testing purposes only)
+DEBUG: false
+# if true, will automatically login a user with username "test"
+# WARNING: DO NOT ENABLE IN PRODUCTION (for testing purposes only)
+MOCK_AUTH: false
+# if true, will fake a successful login response from Google in /login/google
+#     NOTE: this will also modify the behavior of /link/google endpoints
+# WARNING: DO NOT ENABLE IN PRODUCTION (for testing purposes only)
+# will login as the username set in cookie DEV_LOGIN_COOKIE_NAME
+MOCK_GOOGLE_AUTH: false
+
+# if true, will ignore anything configured in STORAGE_CREDENTIALS
+MOCK_STORAGE: true
+# set if you want browsers to only send cookies with requests over HTTPS
+SESSION_COOKIE_SECURE: true
+
+ENABLE_CSRF_PROTECTION: true
+
+# fence (at the moment) attempts a migration on startup. setting this to false will disable that
+# WARNING: ONLY set to false if you do NOT want to automatically migrate your database.
+#          You should be careful about incompatible versions of your db schema with what
+#          fence expects. In other words, things could be broken if you update to a later
+#          fence that expects a schema your database isn't migrated to.
+# NOTE: We are working to improve the migration process in the near future
+ENABLE_DB_MIGRATION: true
```

```suggestion
ENABLE_DB_MIGRATION: false
```
sorry, i misread your message earlier, we want all commons to have `ENABLE_DB_MIGRATION: false` going forward